lib/spack/llnl/util/cpu/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from .microarchitecture import MicroArchitecture, UnsupportedMicroArchitecture
+from .microarchitecture import targets, generic_microarchitecture
+from .detect import host
+
+__all__ = [
+    'MicroArchitecture',
```

Comment:
This should be `Microarchitecture` to be consistent (it's one word).

`microarchitecture` : `micro_architecture` :: `Microarchitecture` : `MicroArchitecture`

Prefer the first ones in this analogy 😄.

Reply:
Done